run/image-processing/README.md

```diff
@@ -0,0 +1,29 @@
+# Cloud Run Image Processing Sample
+
+This sample service applies [Cloud Storage](https://cloud.google.com/storage/docs)-triggered image processing with [Cloud Vision API](https://cloud.google.com/vision/docs) analysis and ImageMagick transformation.
+
+Use it with the [Image Processing with Cloud Run tutorial](http://cloud.google.com/run/docs/tutorials/image-processing).
+
+For more details on how to work with this sample read the [Google Cloud Run Node.js Samples README](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/run).
+
+## Dependencies
+
+* **express**: Web server framework
+* **body-parser**: express middleware for request payload processing
+
+## Environment Variables
```

Comment:
Would it makes sense to explain how to set environment variables?

Reply:
How standalone should the README be from the tutorial?